src/Rest.js

```diff
@@ -16,14 +16,22 @@ define([
 		__PutDirectives = declare(Store.PutDirectives, __HeaderOptions),
 	=====*/
 
-	return declare(Request, {
+	return declare(Request.default, {
 
 		// stringify: Function
 		//		This function performs the serialization of the data for requests to the server. This
 		//		defaults to JSON, but other formats can be serialized by providing an alternate
 		//		stringify function. If you do want to use an alternate format, you will probably
 		//		want to use an alternate parse function for the parsing of data as well.
-		stringify: JSON.stringify,
+		stringify: null,
+
+		constructor: function() {
```

Comment:
Maybe this comment is already moot since you may already be working on this, but otherwise maybe this should have a TODO since I imagine it's restructured to work around how TS initializes properties.